packages/base-styles/_mixins.scss

```diff
@@ -363,46 +367,49 @@
 }
 
 @mixin radio-control {
-	@include input-control;
-	border: $border-width solid $gray-900;
-	margin-right: $grid-unit-15;
+	border: variables.$border-width solid colors.$gray-900;
+	margin-right: variables.$grid-unit-15;
 	transition: none;
-	border-radius: $radius-round;
-	width: $radio-input-size-sm;
-	height: $radio-input-size-sm;
-	min-width: $radio-input-size-sm;
-	max-width: $radio-input-size-sm;
+	border-radius: variables.$radius-round;
+	width: variables.$radio-input-size-sm;
+	height: variables.$radio-input-size-sm;
+	min-width: variables.$radio-input-size-sm;
+	max-width: variables.$radio-input-size-sm;
 	position: relative;
 
+	@media not (prefers-reduced-motion) {
+		transition: box-shadow 0.1s linear;
+	}
```

Comment:
Simply moving `@include input-control;` after the declaration resulted in a visual problem in the `RadioControl` component, as shown below:

![image](https://github.com/user-attachments/assets/adcb1024-6583-408c-8825-0123a5f31f9b)

This is because the round radius is overridden here: https://github.com/WordPress/gutenberg/blob/056bdaf9cc483ad6a41dc4d01cfb283ddb4e8a4e/packages/base-styles/_mixins.scss#L149

Therefore, here I have removed the `@include input-control;` mixin itself and hard-coded only the styles that are truly necessary for the radio control.